app/services/bulk_upload/processor.rb

```diff
@@ -1,12 +1,27 @@
 class BulkUpload::Processor
   attr_reader :bulk_upload
 
+  BLANK_TEMPLATE_ERRORS = [
+    I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base.blank_file"),
+    I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.blank_file"),
+  ].freeze
+
+  WRONG_TEMPLATE_ERRORS = [
+    I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base.wrong_field_numbers_count"),
+    I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base.over_max_column_count"),
+    I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base.wrong_template"),
+    I18n.t("activemodel.errors.models.bulk_upload/lettings/validator.attributes.base.no_headers"),
+    I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.wrong_field_numbers_count"),
+    I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.over_max_column_count"),
+    I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.wrong_template"),
+    I18n.t("activemodel.errors.models.bulk_upload/sales/validator.attributes.base.no_headers"),
+  ].freeze
```

Comment:
Would it make sense to put all the wrong template errors under the same key and then get them all instead of listing them out?
From quick google something like this would let you access them all, but maybe there's a nicer solution:
`I18n.backend.send(:translations)[:en][:activemodel][:errors][:models]...`